### ISSUE TITLE
Hi. I got the problem when I run python DTA_train.py

### ISSUE THREAD

**Issue by sheisxy** [zhaoqichang/AttentionDTA_BIBM#2]
InvalidArgumentError (see above for traceback): indices[56,57] = 66 is not in [0, 65)
	 [[Node: smi_embedding/embedding_lookup = GatherV2[Taxis=DT_INT32, Tindices=DT_INT32, Tparams=DT_FLOAT, _class=["loc:@train_step/Adam/update_smi_embedding/smi_word_embedding/AssignSub"], _device="/job:localhost/replica:0/task:0/device:CPU:0"](smi_embedding/smi_word_embedding/read, input/IteratorGetNext, smi_embedding/embedding_lookup/axis)]]

**Comment by zhaoqichang** [comment]
please show me which line in DTA_train.py causes this issue.

**Comment by sheisxy** [comment]
def inference(smi_tensor, pro_tensor, regularizer=None,
              keep_prob=1, trainlabel=False):
    with tf.variable_scope('smi_embedding', reuse=tf.AUTO_REUSE):
        smi_wordembedding = tf.get_variable(
            "smi_word_embedding", [SMI_DIM, EMBEDDING_DIM])

        smi_embedding = tf.nn.embedding_lookup(smi_wordembedding, smi_tensor)
        pro_wordembedding = tf.get_variable(
            "pro_word_embedding", [PRO_DIM, EMBEDDING_DIM])
        pro_embedding = tf.nn.embedding_lookup(pro_wordembedding, pro_tensor)


this line: 
 smi_embedding = tf.nn.embedding_lookup(smi_wordembedding, smi_tensor)

Do I need some parameters setting ?
Thank you so much !

**Comment by zhaoqichang** [comment]
Please set the SMI_DIM = 67 in DTA_model.py

**Comment by sheisxy** [comment]
Thank you so much!
